src/civis/io/_files.py

```diff
@@ -196,7 +187,7 @@ def _upload_part_base(item, file_path, part_size, file_size):
     return file_response.id
 
 
-def file_to_civis(buf, name=None, client=None, **kwargs):
+def file_to_civis(buf, name=None, expires_at="DEFAULT", description=None, client=None):
```

Comment:
The `**kwargs` before this PR would only accept the `expires_at` key. It's clear by now that we don't really add new features to Civis files that often, so I'm switching to explicitly listing the kwargs here (currently only `expires_at` plus the new `description`) one by one. This update doesn't break existing code calling this function (unless someone has specified the `client` kwarg as a _positional_ arg? seems unlikely, since we always recommend using the kwarg syntax of `client=client` to specify `client`). It also has the advantage of making the function signatures consistent between `file_to_civis`, `dataframe_to_file`, and `json_to_file`.